### ISSUE TITLE
foo.apps-crc.testing resolved to [23.221.222.250] but 192.168.130.11 was expected  - In Centos 7.9

### ISSUE THREAD

**Issue by jkaveripakam** [code-ready/crc#2836]
### General information

  * OS: Linux / macOS / Windows
  * Hypervisor: KVM 
  * Did you run `crc setup` before starting it (Yes/No)? Yes
  * Running CRC on: Laptop / Baremetal-Server / VM  - On window 2008 r server installed VM of Centos 7.9

## CRC version
```bash
# Put `crc version` output here
CodeReady Containers version: 1.35.0+751824a9
OpenShift version: 4.9.5 (embedded in executable)
  
## CRC status
```bash
# Put `crc status --log-level debug` output here

[admin@master ~]$ crc status --log-level debug
DEBU CodeReady Containers version: 1.35.0+751824a9
DEBU OpenShift version: 4.9.5 (embedded in executable)
DEBU Running 'crc status'
DEBU Checking file: /home/admin/.crc/machines/crc/.crc-exist
DEBU Checking file: /home/admin/.crc/machines/crc/.crc-exist
DEBU Found binary path at /home/admin/.crc/bin/crc-driver-libvirt
DEBU Launching plugin server for driver libvirt
DEBU Plugin server listening at address 127.0.0.1:40225
DEBU () Calling .GetVersion
DEBU Using API Version 1
DEBU () Calling .SetConfigRaw
DEBU () Calling .GetMachineName
DEBU (crc) Calling .GetState
DEBU (crc) DBG | time="2021-11-12T13:52:24-05:00" level=debug msg="Getting current state..."
DEBU (crc) DBG | time="2021-11-12T13:52:24-05:00" level=debug msg="Fetching VM..."
DEBU (crc) Calling .GetBundleName
DEBU (crc) Calling .GetIP
DEBU (crc) DBG | time="2021-11-12T13:52:24-05:00" level=debug msg="GetIP called for crc"
DEBU (crc) DBG | time="2021-11-12T13:52:24-05:00" level=debug msg="Getting current state..."
DEBU (crc) DBG | time="2021-11-12T13:52:24-05:00" level=debug msg="IP address: 192.168.130.11"
DEBU Running SSH command: df -B1 --output=size,used,target /sysroot | tail -1
DEBU Using ssh private keys: [/home/admin/.crc/machines/crc/id_ecdsa /home/admin/.crc/cache/crc_libvirt_4.9.5/id_ecdsa_crc]
DEBU SSH command results: err: <nil>, output: 32737570816 13358632960 /sysroot
DEBU cannot get OpenShift status: stat /home/admin/.crc/machines/crc/kubeconfig: no such file or directory
DEBU Making call to close driver server
DEBU (crc) Calling .Close
DEBU Successfully made call to close driver server
DEBU (crc) DBG | time="2021-11-12T13:52:25-05:00" level=debug msg="Closing plugin on server side"
DEBU Making call to close connection to plugin binary
CRC VM:          Running
OpenShift:       Unreachable (v4.9.5)
Disk Usage:      13.36GB of 32.74GB (Inside the CRC VM)
Cache Usage:     16.75GB
Cache Directory: /home/admin/.crc/cache


## CRC config
```bash
# Put `crc config view` output here
[admin@master ~]$ crc config view
- consent-telemetry                     : yes
- memory                                : 12000


## Host Operating System
```bash
# Put the output of `cat /etc/os-release` in case of Linux
# put the output of `sw_vers` in case of Mac
# Put the output of `systeminfo` in case of Windows
[admin@master ~]$ cat /etc/os-release
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"


### Steps to reproduce

  1. cvc setup  - Working fin
  2. crc start -p ./pull-secret --log-level debug 
  3. 
  4. 

### Expected


### Actual


### Logs
[admin@master ~]$ crc setup
INFO Checking if running as non-root
INFO Checking if running inside WSL2
INFO Checking if crc-admin-helper executable is cached
INFO Checking for obsolete admin-helper executable
INFO Checking if running on a supported CPU architecture
INFO Checking minimum RAM requirements
INFO Checking if crc executable symlink exists
INFO Creating symlink for crc executable
INFO Checking if Virtualization is enabled
INFO Checking if KVM is enabled
INFO Checking if libvirt is installed
INFO Checking if user is part of libvirt group
INFO Checking if active user/process is currently part of the libvirt group
INFO Checking if libvirt daemon is running
INFO Checking if a supported libvirt version is installed
INFO Checking if crc-driver-libvirt is installed
INFO Checking if systemd-networkd is running
INFO Checking if NetworkManager is installed
INFO Checking if NetworkManager service is running
INFO Checking if dnsmasq configurations file exist for NetworkManager
INFO Checking if the systemd-resolved service is running
INFO Checking if /etc/NetworkManager/dispatcher.d/99-crc.sh exists
INFO Writing NetworkManager dispatcher file for crc
INFO Using root access: Writing NetworkManager configuration to /etc/NetworkManager/dispatcher.d/99-crc.sh
INFO Using root access: Changing permissions for /etc/NetworkManager/dispatcher.d/99-crc.sh to 755
INFO Using root access: Executing systemctl daemon-reload command
INFO Using root access: Executing systemctl reload NetworkManager
INFO Checking if libvirt 'crc' network is available
INFO Setting up libvirt 'crc' network
INFO Checking if libvirt 'crc' network is active
INFO Starting libvirt 'crc' network
INFO Checking if CRC bundle is extracted in '$HOME/.crc'
INFO Checking if /home/admin/.crc/cache/crc_libvirt_4.9.5.crcbundle exists
Your system is correctly setup for using CodeReady Containers, you can now run 'crc start' to start the OpenShift cluster
[admin@master ~]$ crc start -p ./pull-secret --log-level debug
DEBU CodeReady Containers version: 1.35.0+751824a9
DEBU OpenShift version: 4.9.5 (embedded in executable)
DEBU Running 'crc start'
DEBU Total memory of system is 14432948224 bytes
DEBU No new version available. The latest version is 1.35.0
DEBU Checking if systemd-resolved.service is running
DEBU Running 'systemctl status systemd-resolved.service'
DEBU systemd-resolved.service is already running
INFO Checking if running as non-root
INFO Checking if running inside WSL2
INFO Checking if crc-admin-helper executable is cached
DEBU Running '/home/admin/.crc/bin/crc-admin-helper-linux --version'
DEBU Found crc-admin-helper-linux version v0.0.7
DEBU crc-admin-helper executable already cached
INFO Checking for obsolete admin-helper executable
DEBU Checking if an older admin-helper executable is installed
DEBU No older admin-helper executable found
INFO Checking if running on a supported CPU architecture
INFO Checking minimum RAM requirements
DEBU Total memory of system is 14432948224 bytes
INFO Checking if crc executable symlink exists
INFO Checking if Virtualization is enabled
DEBU Checking if the vmx/svm flags are present in /proc/cpuinfo
DEBU CPU virtualization flags are good
INFO Checking if KVM is enabled
DEBU Checking if /dev/kvm exists
DEBU /dev/kvm was found
INFO Checking if libvirt is installed
DEBU Checking if 'virsh' is available
DEBU 'virsh' was found in /usr/bin/virsh
DEBU Checking 'virsh capabilities' for libvirtd/qemu availability
DEBU Running 'virsh --readonly --connect qemu:///system capabilities'
DEBU Found x86_64 hypervisor with 'hvm' capabilities
INFO Checking if user is part of libvirt group
DEBU Checking if current user is part of the libvirt group
DEBU Current user is already in the libvirt group
INFO Checking if active user/process is currently part of the libvirt group
INFO Checking if libvirt daemon is running
DEBU Checking if libvirtd service is running
DEBU Running 'systemctl status virtqemud.socket'
DEBU Command failed: exit status 4
DEBU stdout:
DEBU stderr: Unit virtqemud.socket could not be found.
DEBU virtqemud.socket is neither running nor listening
DEBU Running 'systemctl status libvirtd.socket'
DEBU Command failed: exit status 4
DEBU stdout:
DEBU stderr: Unit libvirtd.socket could not be found.
DEBU libvirtd.socket is neither running nor listening
DEBU Running 'systemctl status virtqemud.service'
DEBU Command failed: exit status 4
DEBU stdout:
DEBU stderr: Unit virtqemud.service could not be found.
DEBU virtqemud.service is neither running nor listening
DEBU Running 'systemctl status libvirtd.service'
DEBU libvirtd.service is running
INFO Checking if a supported libvirt version is installed
DEBU Checking if libvirt version is >=3.4.0
DEBU Running 'virsh -v'
INFO Checking if crc-driver-libvirt is installed
DEBU Checking if crc-driver-libvirt is installed
DEBU Running '/home/admin/.crc/bin/crc-driver-libvirt version'
DEBU Found crc-driver-libvirt version 0.13.1
DEBU crc-driver-libvirt is already installed
INFO Checking if systemd-networkd is running
DEBU Checking if systemd-networkd.service is running
DEBU Running 'systemctl status systemd-networkd.service'
DEBU Command failed: exit status 4
DEBU stdout:
DEBU stderr: Unit systemd-networkd.service could not be found.
DEBU systemd-networkd.service is not running
INFO Checking if NetworkManager is installed
DEBU Checking if 'nmcli' is available
DEBU 'nmcli' was found in /usr/bin/nmcli
INFO Checking if NetworkManager service is running
DEBU Checking if NetworkManager.service is running
DEBU Running 'systemctl status NetworkManager.service'
DEBU NetworkManager.service is already running
INFO Checking if dnsmasq configurations file exist for NetworkManager
INFO Checking if the systemd-resolved service is running
DEBU Checking if systemd-resolved.service is running
DEBU Running 'systemctl status systemd-resolved.service'
DEBU systemd-resolved.service is already running
INFO Checking if /etc/NetworkManager/dispatcher.d/99-crc.sh exists
DEBU Checking NetworkManager dispatcher file for crc network
DEBU Dispatcher file has the expected content
INFO Checking if libvirt 'crc' network is available
DEBU Checking if libvirt 'crc' network exists
DEBU Running 'virsh --connect qemu:///system net-info crc'
DEBU Checking if libvirt 'crc' definition is up to date
DEBU Running 'virsh --connect qemu:///system net-dumpxml --inactive crc'
DEBU libvirt 'crc' network has the expected value
INFO Checking if libvirt 'crc' network is active
DEBU Checking if libvirt 'crc' network is active
DEBU Running 'virsh --connect qemu:///system net-info crc'
DEBU libvirt 'crc' network is already active
DEBU Checking file: /home/admin/.crc/machines/crc/.crc-exist
DEBU Using secret from configuration
INFO Loading bundle: crc_libvirt_4.9.5...
INFO Creating CodeReady Containers VM for OpenShift 4.9.5...
DEBU Found binary path at /home/admin/.crc/bin/crc-driver-libvirt
DEBU Launching plugin server for driver libvirt
DEBU Plugin server listening at address 127.0.0.1:43133
DEBU () Calling .GetVersion
DEBU Using API Version 1
DEBU () Calling .SetConfigRaw
DEBU () Calling .GetMachineName
DEBU (crc) Calling .GetMachineName
DEBU (crc) Calling .DriverName
DEBU Running pre-create checks...
DEBU (crc) Calling .PreCreateCheck
DEBU (crc) DBG | time="2021-11-12T13:19:35-05:00" level=debug msg="About to check libvirt version"
DEBU (crc) DBG | time="2021-11-12T13:19:35-05:00" level=debug msg="Validating network"
DEBU (crc) DBG | time="2021-11-12T13:19:35-05:00" level=debug msg="Validating storage pool"
DEBU (crc) DBG | time="2021-11-12T13:19:35-05:00" level=debug msg="Could not find storage pool 'crc', trying to create it"
DEBU (crc) DBG | time="2021-11-12T13:19:35-05:00" level=debug msg="Creating storage pool"
DEBU (crc) DBG | time="2021-11-12T13:19:35-05:00" level=info msg="Creating storage pool with XML <pool type=\"dir\">\n  <name>crc</name>\n  <target>\n    <path>/home/admin/.crc/machines/crc</path>\n                                       </target>\n</pool>"
DEBU (crc) DBG | time="2021-11-12T13:19:35-05:00" level=debug msg="Activating pool 'crc'"
DEBU (crc) Calling .GetConfigRaw
DEBU Creating machine...
DEBU (crc) Calling .Create
DEBU (crc) DBG | time="2021-11-12T13:19:35-05:00" level=debug msg="Preparing /home/admin/.crc/machines/crc/crc.qcow2 for machine use"
DEBU (crc) DBG | time="2021-11-12T13:19:35-05:00" level=debug msg="image creation took 43.513948ms"
DEBU (crc) DBG | time="2021-11-12T13:19:35-05:00" level=debug msg="Verifying executable bit set on /home/admin/.crc/machines/crc"
DEBU (crc) DBG | time="2021-11-12T13:19:35-05:00" level=debug msg="Verifying executable bit set on /home/admin/.crc/machines"
DEBU (crc) DBG | time="2021-11-12T13:19:35-05:00" level=debug msg="Verifying executable bit set on /home/admin/.crc"
DEBU (crc) DBG | time="2021-11-12T13:19:35-05:00" level=debug msg="Verifying executable bit set on /home/admin"
DEBU (crc) DBG | time="2021-11-12T13:19:35-05:00" level=debug msg="Verifying executable bit set on /home"
DEBU (crc) DBG | time="2021-11-12T13:19:35-05:00" level=debug msg="Defining VM..."
DEBU (crc) DBG | time="2021-11-12T13:19:35-05:00" level=debug msg="Found x86_64 hypervisor with 'hvm' capabilities"
DEBU (crc) DBG | time="2021-11-12T13:19:35-05:00" level=debug msg="No q35 machine type"
DEBU (crc) DBG | time="2021-11-12T13:19:35-05:00" level=debug msg="disk image capacity is already 33285996544 bytes"
INFO Generating new SSH Key pair...
INFO Generating new password for the kubeadmin user
DEBU Created /home/admin/.crc/machines/crc/.crc-exist
DEBU Machine successfully created
DEBU Found binary path at /home/admin/.crc/bin/crc-driver-libvirt
DEBU Launching plugin server for driver libvirt
DEBU Plugin server listening at address 127.0.0.1:38308
DEBU () Calling .GetVersion
DEBU Using API Version 1
DEBU () Calling .SetConfigRaw
DEBU () Calling .GetMachineName
DEBU (crc) Calling .GetBundleName
DEBU (crc) Calling .GetState
DEBU (crc) DBG | time="2021-11-12T13:19:35-05:00" level=debug msg="Getting current state..."
DEBU (crc) DBG | time="2021-11-12T13:19:35-05:00" level=debug msg="Fetching VM..."
INFO Starting CodeReady Containers VM for OpenShift 4.9.5...
DEBU Updating CRC VM configuration
DEBU (crc) Calling .GetConfigRaw
DEBU (crc) Calling .Start
DEBU (crc) DBG | time="2021-11-12T13:19:35-05:00" level=debug msg="Starting VM crc"
DEBU (crc) DBG | time="2021-11-12T13:19:35-05:00" level=debug msg="Validating network"
DEBU (crc) DBG | time="2021-11-12T13:19:35-05:00" level=debug msg="Validating storage pool"
DEBU (crc) DBG | time="2021-11-12T13:19:41-05:00" level=debug msg="GetIP called for crc"
DEBU (crc) DBG | time="2021-11-12T13:19:41-05:00" level=debug msg="Getting current state..."
DEBU (crc) DBG | time="2021-11-12T13:19:41-05:00" level=debug msg="Waiting for machine to come up 0/60"
DEBU (crc) DBG | time="2021-11-12T13:19:44-05:00" level=debug msg="GetIP called for crc"
DEBU (crc) DBG | time="2021-11-12T13:19:44-05:00" level=debug msg="Getting current state..."
DEBU (crc) DBG | time="2021-11-12T13:19:44-05:00" level=debug msg="Waiting for machine to come up 1/60"
DEBU (crc) DBG | time="2021-11-12T13:19:47-05:00" level=debug msg="GetIP called for crc"
DEBU (crc) DBG | time="2021-11-12T13:19:47-05:00" level=debug msg="Getting current state..."
DEBU (crc) DBG | time="2021-11-12T13:19:47-05:00" level=debug msg="Waiting for machine to come up 2/60"
DEBU (crc) DBG | time="2021-11-12T13:19:50-05:00" level=debug msg="GetIP called for crc"
DEBU (crc) DBG | time="2021-11-12T13:19:50-05:00" level=debug msg="Getting current state..."
DEBU (crc) DBG | time="2021-11-12T13:19:50-05:00" level=debug msg="Waiting for machine to come up 3/60"
DEBU (crc) DBG | time="2021-11-12T13:19:53-05:00" level=debug msg="GetIP called for crc"
DEBU (crc) DBG | time="2021-11-12T13:19:53-05:00" level=debug msg="Getting current state..."
DEBU (crc) DBG | time="2021-11-12T13:19:53-05:00" level=debug msg="Waiting for machine to come up 4/60"
DEBU (crc) DBG | time="2021-11-12T13:19:56-05:00" level=debug msg="GetIP called for crc"
DEBU (crc) DBG | time="2021-11-12T13:19:56-05:00" level=debug msg="Getting current state..."
DEBU (crc) DBG | time="2021-11-12T13:19:56-05:00" level=debug msg="Waiting for machine to come up 5/60"
DEBU (crc) DBG | time="2021-11-12T13:19:59-05:00" level=debug msg="GetIP called for crc"
DEBU (crc) DBG | time="2021-11-12T13:19:59-05:00" level=debug msg="Getting current state..."
DEBU (crc) DBG | time="2021-11-12T13:19:59-05:00" level=debug msg="Waiting for machine to come up 6/60"
DEBU (crc) DBG | time="2021-11-12T13:20:02-05:00" level=debug msg="GetIP called for crc"
DEBU (crc) DBG | time="2021-11-12T13:20:02-05:00" level=debug msg="Getting current state..."
DEBU (crc) DBG | time="2021-11-12T13:20:02-05:00" level=debug msg="Waiting for machine to come up 7/60"
DEBU (crc) DBG | time="2021-11-12T13:20:05-05:00" level=debug msg="GetIP called for crc"
DEBU (crc) DBG | time="2021-11-12T13:20:05-05:00" level=debug msg="Getting current state..."
DEBU (crc) DBG | time="2021-11-12T13:20:05-05:00" level=debug msg="Waiting for machine to come up 8/60"
DEBU (crc) DBG | time="2021-11-12T13:20:08-05:00" level=debug msg="GetIP called for crc"
DEBU (crc) DBG | time="2021-11-12T13:20:08-05:00" level=debug msg="Getting current state..."
DEBU (crc) DBG | time="2021-11-12T13:20:08-05:00" level=debug msg="Waiting for machine to come up 9/60"
DEBU (crc) DBG | time="2021-11-12T13:20:11-05:00" level=debug msg="GetIP called for crc"
DEBU (crc) DBG | time="2021-11-12T13:20:11-05:00" level=debug msg="Getting current state..."
DEBU (crc) DBG | time="2021-11-12T13:20:11-05:00" level=debug msg="Waiting for machine to come up 10/60"
DEBU (crc) DBG | time="2021-11-12T13:20:14-05:00" level=debug msg="GetIP called for crc"
DEBU (crc) DBG | time="2021-11-12T13:20:14-05:00" level=debug msg="Getting current state..."
DEBU (crc) DBG | time="2021-11-12T13:20:14-05:00" level=debug msg="Waiting for machine to come up 11/60"
DEBU (crc) DBG | time="2021-11-12T13:20:17-05:00" level=debug msg="GetIP called for crc"
DEBU (crc) DBG | time="2021-11-12T13:20:17-05:00" level=debug msg="Getting current state..."
DEBU (crc) DBG | time="2021-11-12T13:20:17-05:00" level=debug msg="Waiting for machine to come up 12/60"
DEBU (crc) DBG | time="2021-11-12T13:20:20-05:00" level=debug msg="GetIP called for crc"
DEBU (crc) DBG | time="2021-11-12T13:20:20-05:00" level=debug msg="Getting current state..."
DEBU (crc) DBG | time="2021-11-12T13:20:20-05:00" level=debug msg="Waiting for machine to come up 13/60"
DEBU (crc) DBG | time="2021-11-12T13:20:23-05:00" level=debug msg="GetIP called for crc"
DEBU (crc) DBG | time="2021-11-12T13:20:23-05:00" level=debug msg="Getting current state..."
DEBU (crc) DBG | time="2021-11-12T13:20:23-05:00" level=debug msg="IP address: 192.168.130.11"
DEBU (crc) DBG | time="2021-11-12T13:20:23-05:00" level=info msg="Found IP for machine: 192.168.130.11"
DEBU (crc) Calling .GetConfigRaw
DEBU Waiting for machine to be running, this may take a few minutes...
DEBU retry loop: attempt 0
DEBU (crc) Calling .GetState
DEBU (crc) DBG | time="2021-11-12T13:20:23-05:00" level=debug msg="Getting current state..."
DEBU Machine is up and running!
DEBU (crc) Calling .GetState
DEBU (crc) DBG | time="2021-11-12T13:20:23-05:00" level=debug msg="Getting current state..."
DEBU (crc) Calling .GetIP
DEBU (crc) DBG | time="2021-11-12T13:20:23-05:00" level=debug msg="GetIP called for crc"
DEBU (crc) DBG | time="2021-11-12T13:20:23-05:00" level=debug msg="Getting current state..."
DEBU (crc) DBG | time="2021-11-12T13:20:23-05:00" level=debug msg="IP address: 192.168.130.11"
INFO CodeReady Containers instance is running with IP 192.168.130.11
DEBU Waiting until ssh is available
DEBU retry loop: attempt 0
DEBU Running SSH command: exit 0
DEBU Using ssh private keys: [/home/admin/.crc/cache/crc_libvirt_4.9.5/id_ecdsa_crc /home/admin/.crc/machines/crc/id_ecdsa]
DEBU SSH command results: err: <nil>, output:
INFO CodeReady Containers VM is running
DEBU Running SSH command: cat /home/core/.ssh/authorized_keys
DEBU SSH command results: err: <nil>, output: ecdsa-sha2-nistp521 AAAAE2VjZHNhLXNoYTItbmlzdHA1MjEAAAAIbmlzdHA1MjEAAACFBAFhGyl5EFZXkQJvyqs1oL6+YoauefhAV2CxzXtJmL9unz7LXCzWxV9XRnGYPWgx9ROwiiLeUlNoWy6ZCd                                     utBfYXvwB2POddt1ah6FMN0BSOyhSwypO2Tb3UYT6PRA7aVezTiuWijnigUDftgD62J/Q/wVIA+h7b/ddbZAslI+2ihqJE7g== core
INFO Updating authorized keys...
DEBU Creating /home/core/.ssh/authorized_keys with permissions 0644 in the CRC VM
DEBU Running SSH command: <hidden>
DEBU SSH command succeeded
DEBU Running SSH command: realpath /dev/disk/by-label/root
DEBU SSH command results: err: <nil>, output: /dev/vda4
DEBU Using root access: Growing /dev/vda4 partition
DEBU Running SSH command: sudo /usr/bin/growpart /dev/vda 4
DEBU SSH command results: err: Process exited with status 1, output: NOCHANGE: partition 4 is size 63961055. it cannot be grown
DEBU No free space after /dev/vda4, nothing to do
DEBU Using root access: make root Podman socket accessible
DEBU Running SSH command: sudo chmod 777 /run/podman/ /run/podman/podman.sock
DEBU SSH command results: err: <nil>, output:
DEBU Creating /var/srv/dnsmasq.conf with permissions 0644 in the CRC VM
DEBU Running SSH command: <hidden>
DEBU SSH command succeeded
DEBU Running SSH command: systemctl status crc-dnsmasq.service
DEBU SSH command results: err: Process exited with status 3, output: ● crc-dnsmasq.service - Podman container-9e6a08b6ae6b7e9bc386a6efa0d46a6049532e2db42f5cfc30252f6ad0f88236.service
   Loaded: loaded (/etc/systemd/system/crc-dnsmasq.service; disabled; vendor preset: disabled)
   Active: inactive (dead)
     Docs: man:podman-generate-systemd(1)
DEBU Using root access: Executing systemctl enable crc-dnsmasq.service
DEBU Running SSH command: sudo systemctl enable crc-dnsmasq.service
DEBU SSH command results: err: <nil>, output:
DEBU Using root access: Executing systemctl daemon-reload command
DEBU Running SSH command: sudo systemctl daemon-reload
DEBU SSH command results: err: <nil>, output:
DEBU Using root access: Executing systemctl start crc-dnsmasq.service
DEBU Running SSH command: sudo systemctl start crc-dnsmasq.service
DEBU SSH command results: err: <nil>, output:
DEBU Running '/home/admin/.crc/bin/crc-admin-helper-linux rm api.crc.testing oauth-openshift.apps-crc.testing console-openshift-console.apps-crc.testing downloads-openshift-console.apps-crc.testing ca                                     nary-openshift-ingress-canary.apps-crc.testing default-route-openshift-image-registry.apps-crc.testing'
DEBU Running '/home/admin/.crc/bin/crc-admin-helper-linux add 192.168.130.11 api.crc.testing oauth-openshift.apps-crc.testing console-openshift-console.apps-crc.testing downloads-openshift-console.app                                     s-crc.testing canary-openshift-ingress-canary.apps-crc.testing default-route-openshift-image-registry.apps-crc.testing'
DEBU Running SSH command: cat /etc/resolv.conf
DEBU SSH command results: err: <nil>, output: # Generated by NetworkManager
nameserver 192.168.130.1
DEBU Creating /etc/resolv.conf with permissions 0644 in the CRC VM
DEBU Running SSH command: <hidden>
DEBU SSH command succeeded
DEBU retry loop: attempt 0
DEBU Running SSH command: host -R 3 foo.apps-crc.testing
DEBU SSH command results: err: <nil>, output: foo.apps-crc.testing has address 192.168.130.11
INFO Check internal and public DNS query...
DEBU Running SSH command: curl --head quay.io
DEBU SSH command results: err: <nil>, output: HTTP/1.1 301 Moved Permanently
Server: nginx/1.12.1
Date: Fri, 12 Nov 2021 18:20:32 GMT
Content-Type: text/html
Content-Length: 185
Connection: keep-alive
Location: https://quay.io/

INFO Check DNS query from host...
DEBU api.crc.testing resolved to [192.168.130.11]
DEBU foo.apps-crc.testing resolved to [23.221.222.250]
WARN foo.apps-crc.testing resolved to [23.221.222.250] but 192.168.130.11 was expected
DEBU Making call to close driver server
DEBU (crc) Calling .Close
DEBU Successfully made call to close driver server
DEBU Making call to close connection to plugin binary
DEBU (crc) DBG | time="2021-11-12T13:20:32-05:00" level=debug msg="Closing plugin on server side"
DEBU Making call to close driver server
DEBU (crc) Calling .Close
DEBU Successfully made call to close driver server
DEBU Making call to close connection to plugin binary
DEBU (crc) DBG | time="2021-11-12T13:20:32-05:00" level=debug msg="Closing plugin on server side"
Failed to query DNS from host: Invalid IP for foo.apps-crc.testing


Before gather the logs try following if that fix your issue
```bash
$ crc delete -f
$ crc cleanup
$ crc setup
$ crc start --log-level debug
```
I have tried all options.. 

Please consider posting the output of `crc start --log-level debug`  on http://gist.github.com/ and post the link in the issue.

**Comment by jkaveripakam** [comment]
I have tried all the below options but getting the same problem.. 

$ crc delete -f
$ crc cleanup
$ crc setup
$ crc start --log-level debug

**Comment by praveenkumar** [comment]
@jkaveripakam This is duplicate of https://github.com/code-ready/crc/issues/1298 one.
